.readthedocs.yaml

```diff
@@ -1,12 +1,35 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
 version: 2
 
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+    # You can also specify other tool versions:
+    # nodejs: "20"
+    # rust: "1.70"
+    # golang: "1.20"
```

Comment:
Kind of a noise. These are documentation notes, we don't need them here.

Reply:
It's the standard from ReadTheDocs. Suggest keeping it in